R/build.r

```diff
@@ -309,6 +309,23 @@
 #'    deploy:
 #'      install_metadata: true
 #'    ```
+#' @section YAML config - footer:
+#' By default, the footer is automatically populated with:
+#' * the names of the authors `authors`, on the left;
```

Comment:
Maybe worth cross-linking to docs on authors?

Reply:
How do you link to a section of the same help file?

Reply:
btw in general I am wondering whether it'd make sense to make all the YAML config sections re-usable Rmd components, to make a big vignette with all the YAML options (I know there's search, but I still remember not easily finding my way).

Reply:
You can't 😢  Least worst thing is to link to section in pkgdown docs

Reply:
:joy: I had re-read https://roxygen2.r-lib.org/articles/rd-formatting.html#links-2 with so much hope